cpu/cc26xx_cc13xx/ldscripts/cc26xx_cc13xx.ld

```diff
@@ -22,7 +22,8 @@ INCLUDE cortexm_rom_offset.ld
 MEMORY
 {
     rom      (rx)   : ORIGIN = _rom_start_addr + _rom_offset, LENGTH = _fw_rom_length
-    ccfg     (rx)   : ORIGIN = 0x00057FA8,                    LENGTH = 88
+    /* CCFG starts at the end of ROM */
+    ccfg     (rx)   : ORIGIN = _rom_length,                   LENGTH = 88
```

Comment:
How about keeping `ROM_LEN` at 128k and then use

```suggestion
    ccfg     (rx)   : ORIGIN = _rom_length - 88,              LENGTH = 88
```

here?

This keeps the config offset handling better contained.

Reply:
Won't it cause `rom` to overlap with `ccfg` :eyes:?

Reply:
Shouldn't the linker warn about overlapping sections? That would then be a build error.

Reply:
>overlaps section seems to be a valid linker error string

TIL :smile: , the test blob on cc26x0 failed succesfully with a 128K blob and the .ccfg section :relieved:

Reply:
Fixed at https://github.com/RIOT-OS/RIOT/pull/15126/commits/ed2e04904b35ba5ca0f72fdbfd5ffc25851c55cc . Thanks for pointing it! Much cleaner